woocommerce/woocommerce-ios
# Updated the Storage layer for introducing `InboxNote` and `InboxAction` entities

Closes #6057 


### Description
This PR updates the Storage layer with two new entities `InboxNote` and `InboxAction` to persist the Inbox Notes. This also includes a one-to-many relationship between Inbox Notes and Inbox Actions.
I also updated the name of the property `isDeleted` in `InboxNote` to `isRemoved` because of a conflict with core data property `isDeleted` in `CoreDataProperties`.

### Testing instructions
Make sure that the app builds and runs properly without crashing with this new update. The Yosemite layer will be updated in a separate PR with these changes.


---
